### PR TITLE
SPRT-94 Advance Flow by Hash Docs

### PR DESCRIPTION
**Pull request by pyr0gan**
- Add a document to explain how to advance a Staking API flow by using a transaction hash for a confirmed transaction.

Screen Shot 2023-02-03 at 10 38 19 AM

### CODE REVIEW

File: docs/guides/staking-api/advance-flows-using-transaction-hash.mdx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+import Link from "@docusaurus/Link";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+## Motivation
+
+The Staking API provides the option for clients to use an existing solution to sign and broadcast transactions,
+and to use a transaction hash from a completed transaction to advance the state of a flow &mdash; in lieu of providing a signed transaction payload.
+
```
Comment (coltonc3) [inline docs/guides/staking-api/advance-flows-using-transaction-hash.mdx:14]:
```suggestion
The Staking API provides the option for clients to use an existing solution to sign and broadcast transactions themselves,
and to use a transaction hash from a completed transaction to advance the state of a flow and confirm the transaction on-chain &mdash; in lieu of providing a signed transaction payload.

```

File: docs/guides/staking-api/advance-flows-using-transaction-hash.mdx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  method: 'PUT',
+  headers: {
+    "Authorization" : "<API_KEY>",
+    "Content-Type": "application/json"
+  },
+  body: JSON.stringify({
+    "name": "confirm_bonding_tx_by_hash",
+    "inputs": {
+        "hash": "0x53b66da426dca5386408f0a3cdc5300477dd7151c7459701e9dcbbf0fa0b07c8",
+        "block_number": "14295923"
```
Comment (coltonc3) [inline docs/guides/staking-api/advance-flows-using-transaction-hash.mdx:123]: can we add a note that `block_number` is only required for Polkadot flows?

Comment (pyr0gan) [reply docs/guides/staking-api/advance-flows-using-transaction-hash.mdx:123]: It is mentioned above,

> In the case of Polkadot, the block number of the block which includes the transaction hash must also be specified.

Is there a clearer way to express this without repetition?

**Review by artiona — COMMENTED** [review]
With Colton's suggestions in, lgtm

**Pull request merged**